qiskit/primitives/base_sampler.py

```diff
@@ -71,17 +71,13 @@
     theta3 = [0, 1, 2, 3, 4, 5, 6, 7]
 
     with Sampler(circuits=[pqc, pqc2], parameters=[pqc.parameters, pqc2.parameters]) as sampler:
-        result = sampler(parameters=[theta1, theta2, theta3], circuits=[0, 0])
+        result1 = sampler([0, 0], [theta1, theta2])
```

Comment:
I recently modified the example to use multiple pqc.

```suggestion
        result = sampler([0, 0, 1], [theta1, theta2, theta3])
```